src/container/MovieContainer.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import MoviePoster from './MoviePoster'
+import MoviePoster from '../poster/MoviePoster'
 import './MovieContainer.css'
```

Comment:
Thanks for updating the file architecture here. This will keep our components more organized moving forward and our hierarchy will be clearer to the instructors and future devs.